packages/gatsby/src/utils/eslint-config.ts

```diff
@@ -5,10 +5,22 @@ import path from "path"
 const eslintRulePaths = path.resolve(`${__dirname}/eslint-rules`)
 const eslintRequirePreset = require.resolve(`./eslint/required`)
 
-export const eslintRequiredConfig: CLIEngine.Options = {
+// eslint-loader options are few eslint-loader specific options and rest of it is passed to
+// eslint's CLIEngine
+type EslintLoaderOptions = CLIEngine.Options & {
+  emitWarning?: boolean
+}
+
+export const eslintRequiredConfig: EslintLoaderOptions = {
   rulePaths: [eslintRulePaths],
   useEslintrc: false,
+  // this forces any errors to be webpack warnings, minimal eslint-loader
+  // should NEVER prevent bundle from building. Our eslint config for it
+  // only contain rules set to warnings, but loader itself can emit errors
+  // if parsing failed etc.
+  emitWarning: true,
   baseConfig: {
+    parser: require.resolve(`babel-eslint`),
```

Comment:
Shouldn't we add it to the package.json?
```suggestion
    parser: require.resolve(`babel-eslint`),
```

Reply:
we already have it - https://github.com/gatsbyjs/gatsby/blob/7b6454b753f1098ee76b21a18cc1f6d16a1fe112/packages/gatsby/package.json#L34